btcpay-server/umbrel-app.yml

```diff
@@ -34,20 +34,24 @@ path: ""
 defaultUsername: ""
 defaultPassword: ""
 releaseNotes: >
-  New features for 1.7.5!
-
-  - Greenfield: Lightning addresses API (#4546) @Kukks
+  Update recommended for shared instances.
 
   
   Bug fixes
 
-  - Fix several HTML injections (#4545) @NicolasDorier
-  - BIP21: Uppercase addresses only in QR, not in payment URL (#4553) @dennisreimann
-  - Checkout v2: UI fixes (#4552) @dennisreimann
+  - Fix second order XSS: Harden file type input (#4635) @NicolasDorier
```

Comment:
@JimSB - a newline is needed between each bullet in order to format the releaseNotes section so that there is one bullet per line

Reply:
added a new commit with new lines! ty!